docker-compose.yml

```diff
@@ -43,6 +45,8 @@ services:
         command: /app/manage.py runserver 0.0.0.0:8678
         restart: unless-stopped
         tty: true
+        # https://docs.docker.com/compose/compose-file/build/#illustrative-example
+        image: ghcr.io/simjanos-dev/linguacafe-python-service:main
         build:
             dockerfile: ./docker/PythonDockerfile
```

Comment:
If users are to use these files as examples for their own it is best to remove the build steps to avoid possible confusion or accidental builds that leave the user with a different version of dependencies than the reference image.